examples/complete/config/cert-manager.yaml

```diff
@@ -5,6 +5,8 @@ crds:
   # This option decides if the CRDs should be installed
   # as part of the Helm installation.
   enabled: true
+  # This option makes it so that the "helm.sh/resource-policy": keep annotation is NOT added to the CRD. This means Helm will delete the CRD when the Helm release is uninstalled.
```

Comment:
This change should be moved to module's config.yaml file. We use example's config files just to override the changes.